src/directives/markersWatchOptions.js

```diff
@@ -1,31 +0,0 @@
-angular.module("leaflet-directive").directive('markersWatchOptions',
```

Comment:
removed in favor of watchOptions.js which creates multiple directive control directives to handle common defaults.